aries-site/src/pages/learn/how-to-easily-add-search-and-filter-to-datatable.mdx

```diff
@@ -0,0 +1,365 @@
+import { Box, Image, Notification } from 'grommet';
+
+<Notification
+  status="info"
+  width="large"
+  margin={{ bottom: 'medium' }}
+  message='This guide uses a DataTable that is configured to fetch data from a REST API. If you’d like to learn more about how this configuration works, check out the guide on "How to fetch data from an API and render it in a DataTable."'
+  actions={[
+    {
+      label: 'Guide TBD',
+      href: '#',
+    },
+  ]}
+/>
+
+Let’s get started!
+
+Here's a [CodeSandbox template](https://codesandbox.io/s/hpe-design-system-template-lrdtpe?file=/index.js) with the required dependencies already installed.
+
+Copy and paste this paginated DataTable code into your CodeSandbox. You can replace all of the code.
+
+```
```

Comment:
Can we truncate the visual space for this code when rendered, requiring the caller to scroll within it if desired? It feels rather long in the rendered page.